README.md

```diff
@@ -53,39 +53,42 @@ brownie run scripts/deploy_pancakeswap.py --network bsc-main
 
 Deposit a token with its amount with an expected token address and amount. This is run by users.
 
-| Key         | Type    | Description                             |
-| ----------- | ------- | --------------------------------------- |
-| token0      | address | Deposit token address                   |
-| token1      | address | Expected token address                  |
-| amount0     | uint256 | Deposit token amount                    |
-| amount1_min | uint256 | Expected token amount for stop loss     |
-| amount1_max | uint256 | Expected token amount for profit taking |
+| Key           | Type      | Description                                 |
+| ------------- | --------- | ------------------------------------------- |
+| path          | address[] | Initial token swap path via Uniswap V2      |
+| amount0       | uint256   | Deposit token amount                        |
+| min_amount1   | uint256   | Expected token amount from the initial swap |
+| profit_taking | uint256   | Permille of profit_taking                   |
```

Comment:
why is it permille and not percent?

Reply:
I want to set profit taking as 5.5% maybe?